eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -469,8 +470,8 @@ public function loadContentByRemoteId($remoteId, array $languages = null, $versi
      * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException if there is a provided remoteId which exists in the system
      *                                                                        or there is no location provided (4.x) or multiple locations
      *                                                                        are under the same parent or if the a field value is not accepted by the field type
-     * @throws \eZ\Publish\API\Repository\Exceptions\ContentFieldValidationException if a field in the $contentCreateStruct is not valid
-     * @throws \eZ\Publish\API\Repository\Exceptions\ContentValidationException if a required field is missing or is set to an empty value
```

Comment:
Yes, I have a task point about doc-ing that. (Don't see a bc way of doing what was requested.)